server/player/playerdb/json.go

```diff
@@ -138,4 +138,5 @@ type jsonEffect struct {
 	Level    int
 	Duration time.Duration
 	Ambient  bool
+	Infinite bool
```

Comment:
Not a problem with this PR, but particle hidden does not appear to be saved? Feel free to fix that